From david-b@pacbell.net Fri Aug 17 22:20:05 2007
From: David Brownell <david-b@pacbell.net>
Date: Fri, 17 Aug 2007 22:19:59 -0700
Subject: USB: ohci, fix oddball gcc warning
To: Greg KH <greg@kroah.com>
Cc: Andrew Morton <akpm@linux-foundation.org>
Message-ID: <200708172220.00318.david-b@pacbell.net>
Content-Disposition: inline


Some versions of GCC recently grew annoying warnings about constants.
This gets rid of that warning from the OHCI driver.

Signed-off-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-dbg.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/ohci-dbg.c
+++ b/drivers/usb/host/ohci-dbg.c
@@ -74,7 +74,7 @@ urb_print (struct urb * urb, char * str,
 
 #define ohci_dbg_sw(ohci, next, size, format, arg...) \
 	do { \
-	if (next) { \
+	if (next != NULL) { \
 		unsigned s_len; \
 		s_len = scnprintf (*next, *size, format, ## arg ); \
 		*size -= s_len; *next += s_len; \
